call-for-adoption-process.md

```diff
@@ -0,0 +1,11 @@
+* this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption
+
+* when the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen
+
+* changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa
+
+* when the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked as up an Editors Draft
+
+* if instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties, with an expectation that the reviews made during the new period would focus on the said changes.
```

Comment:
Nit: confirm the resulting -> confirm the resulting
Style: "with an expectation" -> "The reviews made during the new period should focus on the changes made subsequent to the first CfA."